src/components/registerV2/backupPassphrase.js

```diff
@@ -1,65 +1,16 @@
-import React from 'react';
-import QRCode from 'qrcode.react';
-import { CopyToClipboard } from 'react-copy-to-clipboard';
 import { translate } from 'react-i18next';
-import moment from 'moment';
+import React from 'react';
 import grid from 'flexboxgrid/dist/flexboxgrid.css';
+
```

Comment:
I dont know if this new line was intended or not, but if not maybe you can fix it

Reply:
It was inserted by [import-js](https://github.com/Galooshi/import-js) which I use since a few weeks to automatically generate imports for me. I can recommend the tool for you and @massao too, it has plugins for various editors.

 I changed config not to insert the empty line between internal and external imports https://github.com/LiskHQ/lisk-hub/pull/2142/commits/7fc14b1e61daf3616c3450d032c12e4ff68bd60e